pwa-devdocs/src/pwa-buildpack/troubleshooting/index.md

```diff
@@ -95,7 +95,7 @@ Some users have suggested deleting the `devcert` folder to trigger certificate r
 ${User}\AppData\Local\devcert
   ```
 
-[create an issue]: https://github.com/magento-research/pwa-buildpack/issues
+[create an issue]: https://github.com/magento/pwa-buildpack/issues
```

Comment:
🙅‍♂ This did not get transferred. Original repository "has been archived by the owner. It is now read-only". Do we drop this in favor of creating issues in `pwa-studio` proper?